v2/tools/generator/internal/codegen/pipeline/handle_float_values.go

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package pipeline
+
+import (
+	"context"
+
+	"github.com/Azure/azure-service-operator/v2/tools/generator/internal/astmodel"
+	"github.com/pkg/errors"
+	kerrors "k8s.io/apimachinery/pkg/util/errors"
+)
+
+// HandleFloatValuesStageID is the unique identifier of this stage
+const (
+	HandleFloatValuesStageID = "handleFloatValues"
+)
+
+func HandleFloatValues() *Stage {
```

Comment:
Similarly to below, improve the name of this a bit to be a bit more clear about the specific float values being handled. It's not actually all of them, it's all of them that are validated? So `HandleValidatedFloats` or something?

Reply:
I agree that the name is too generic - even just `TransformValidatedFloats` would be clearer.